apps/req.c

```diff
@@ -1094,10 +1088,10 @@ static int auto_info(X509_REQ *req, STACK_OF(CONF_VALUE) *dn_sk,
          */
         for (p = v->name; *p; p++) {
 #ifndef CHARSET_EBCDIC
-            spec_char = ((*p == ':') || (*p == ',') || (*p == '.'));
+            spec_char = *p == ':' || *p == ',' || *p == '.';
 #else
-            spec_char = ((*p == os_toascii[':']) || (*p == os_toascii[','])
-                    || (*p == os_toascii['.']));
+            spec_char = *p == os_toascii[':'] || *p == os_toascii[',']
+                         || *p == os_toascii['.'];
```

Comment:
Same here.  WIth all the `=` in there, it's easy to get cross-eyed.  The parentheses help.